cdn/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+const webpack = require('webpack')
+const mode = process.env.NODE_ENV || 'production'
+
+module.exports = {
+  output: {
+    filename: `worker.${mode}.js`,
+    path: path.join(__dirname, 'dist'),
+  },
+  devtool: 'source-map',
+  mode,
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js'],
+    plugins: [
+      // XXX this doesn't work right now due to webpack errors
+      // new webpack.DefinePlugin({
+      //   SENTRY_RELEASE: process.env.SENTRY_RELEASE || 'local'
+      // })
```

Comment:
Ahh, now I see why the `SENTRY_RELEASE` stuff is commented out throughout the code, but still in place. Feel free to ignore my removal comments if this is still being worked on.